compiler/rustc_session/src/options.rs

```diff
@@ -58,18 +59,180 @@ macro_rules! hash_substruct {
     };
 }
 
+// For example, when external target modifier is '-Zregparm=2':
+// Target modifier hash (hash_of('regparm')) + user value ('2') from external crate
+// is converted into description: prefix ('Z'), name ('regparm'), tech value ('Some(2)')
+pub struct ExtendedTargetModifierInfo {
+    pub prefix: String,
+    pub name: String,
+    pub tech_value: String,
+}
+
+// A recorded -Zopt_name=opt_value (or -Copt_name=opt_value)
+// for abi-changing or vulnerability-affecting options
```

Comment:
Suggestion: prefer rustdoc comments on these, e.g.

```
/// A recorded `-Zopt_name=opt_value` (or `-Copt_name=opt_value`) for
/// ABI-changing or vulnerability-affecting options.
#[derive(Debug, Clone, Encodable, Decodable)]
pub struct TargetModifier {
    /// Option name hash.
    pub opt: OptionsTargetModifiers,
    /// User-provided option value (before parsing).
    pub value_name: String,
}
```

so that these would show up on https://doc.rust-lang.org/nightly/nightly-rustc/index.html.

Reply:
Fixed.